src/dscigametrics/stat_summary.py

```diff
@@ -21,7 +24,7 @@ def stat_summary(data, campaign_id, start_date, end_date):
     Returns
     -------
     summary_table : pd.DataFrame
-        A pandas dataframe contains the mean, median and standard deviation of the four metrics.
+        A pandas dataframe contains the mean, median and standard deviation of the four metrics of each day.
```

Comment:
shouldn't it calculate metrics from the start date to the end date, not "for each day"?

Reply:
Oh! It's actually:
> 4 metrics of each day between the start date to the end date -> mean, median and standard deviation of these data